include/ofi.h

```diff
@@ -149,8 +149,21 @@ extern "C" {
 		_a > _b ? _a : _b; })
 #endif
 
+#define BIT(i) (1ul << (i))
```

Comment:
BIT is already defined 20~ lines later in the same file.

Reply:
Fixed.

Reply:
Edited. Commented by mistake